x-pack/plugins/fleet/server/services/agents/upgrade.ts

```diff
@@ -89,7 +89,7 @@ export async function sendUpgradeAgentsActions(
     const batchSize = options.batchSize ?? SO_SEARCH_LIMIT;
     const res = await getAgentsByKuery(esClient, soClient, {
       kuery: options.kuery,
-      showInactive: options.showInactive ?? false,
+      showInactive: true,
```

Comment:
Regarding this and @juliaElastic's suggestion in https://github.com/elastic/kibana/issues/171914#issuecomment-1978706640 - this does look like a significant enough refactor unless it's just renaming this option. Note that the options were never including `showInactive`. Same comment for bulk reassign.